domain/src/test/java/org/oppia/android/domain/platformparameter/PlatformParameterSingletonTest.kt

```diff
@@ -0,0 +1,142 @@
+package org.oppia.android.domain.platformparameter
+
+import android.app.Application
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import dagger.BindsInstance
+import dagger.Component
+import javax.inject.Inject
+import javax.inject.Singleton
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.android.app.model.PlatformParameter as ParameterValue
+import org.oppia.android.testing.robolectric.RobolectricModule
+import org.oppia.android.testing.threading.TestDispatcherModule
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.LooperMode
+
+const val stringPlatformParameterName = "string_platform_parameter_name"
```

Comment:
I recommend putting these in the class since they're specific to this class being tested.

Reply:
You mean to say I should put these into PlatformParameterSingleton? If yes then isn't it wrong to put the test specific constants in a Production class. @jcqli

Reply:
Oh I meant below in this test class! Around line 38.

Reply:
Yeah that can be done its just that in previous tests that I worked, they had constant like this outside of the class ([ test file ](https://github.com/ARJUPTA/oppia-android/blob/9a691d448ed7f7eaeeb7471938669c78e8a875ff/app/src/sharedTest/java/org/oppia/android/app/options/ReadingTextSizeFragmentTest.kt#L79)from my earlier PR)

Reply:
That's fair -- I don't think we have any specific conventions on how to do this so I'll leave it up to you. If you do choose to leave as is, they probably should be `private` and should be named `STRING_PLATFORM_PARAMETER_NAME` etc.

Reply:
Okay noted